examples/customized_models/HGT/acm_nc.yaml

```diff
@@ -3,15 +3,12 @@ version: 1.0
 gsf:
   basic:
     model_encoder_type: rgcn
-    graph_name: acm
     backend: gloo
     ip_config: ip_list.txt
     part_config: /data/acm_nc/acm.json
     verbose: false
     mixed_precision: false
     no_validation: false
-    mp_opt_level: O2
-    train_nodes: 10
```

Comment:
These two args are removed from the all the yaml files?

Reply:
I think so. We don't have there two in our argument list.